app/pages/v/[id].js

```diff
@@ -7,6 +7,17 @@ export default function Video({ submission }) {
   return <Layout>
     <Head>
       <title>{ submission.description } - Μεμονωμένο Περιστατικό</title>
+      <meta property="og:locale" content="el_GR" />
+      <meta property="og:title" content={ submission.description + " - Μεμονωμένο Περιστατικό" } />
+      <meta property="og:description" content={ submission.description } />
+      <meta property="og:image" content={ submission.thumbURL? submission.thumbURL: undefined } />
```

Comment:
Maybe show our default social media header (same as in home page) instead of `undefined`.